bin/test_rhel8_integration

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash -ex
+
+cleanup() {
+  echo "Cleaning up..."
+  echo "Cleaning up... logs in cleanup.log"
+  docker-compose rm --stop --force
+}
+
+DEBUG="false"
+if [[ "$1" = "--debug" || "$1" = "-d" ]]; then
+  DEBUG="true"
+  shift
+fi
+
+trap 'echo "ERROR: Test script encountered an error!"; docker-compose logs &> cleanup.log; cleanup' ERR
+trap 'cleanup' EXIT
+cleanup
+
+#echo "Building API container..."
+#docker-compose build
+
+echo "Generating certificate..."
+docker-compose up openssl
+
+echo "Starting Conjur..."
+docker-compose up -d conjur-appliance
+
+
+echo "Configuring Conjur..."
+export ADMIN_PASSWORD=P@S5-ITAbXJlxH1MU
```

Comment:
I think it will be better to generate a key here dynamically so the password will not be exposed to the outside world

Reply:
^ agree